assets/app/scripts/controllers/build.js

```diff
@@ -113,7 +113,7 @@ angular.module('openshiftConsole')
 
       (function initLogs() {
         angular.extend($scope, {
-          logs: [],
+          logs: [{text: 'Logs for '+ $routeParams.build}, {text: ' '}],
```

Comment:
Ah, originally thought it would be handy in the downloadable file.  Still agree, shouldn't be in here.